app/templates/deprecations-toolbar.hbs

```diff
@@ -1,7 +1,14 @@
 <div class="toolbar">
   {{clear-button action="clear" classNames="toolbar__icon-button js-clear-deprecations-btn"}}
 
+  <div class="divider" aria-hidden="true"></div>
+
   <div class="toolbar__search js-deprecations-search">
     {{input value=searchVal placeholder="Search"}}
+    {{#if searchVal}}
+      <button {{action "clearSearchVal"}} class="toolbar__icon-button toolbar__search-clear-button" title="clear">
```

Comment:
It might also be good to put these common clear elements in some common template and use them, rather than repeating ourselves.